internal/backends/qt/qt_accessible.rs

```diff
@@ -616,6 +622,8 @@ cpp! {{
         void *interface_cast(QAccessible::InterfaceType t) override {
             if (t == QAccessible::ValueInterface && !item_string_property(m_data, QAccessible::Value).isEmpty()) {
                 return static_cast<QAccessibleValueInterface*>(this);
+            } else if (t == QAccessible::ActionInterface && !item_string_property(m_data, QAccessible::Value).isEmpty()) {
```

Comment:
Hm, I'm wondering if the value check is correct.

Actions like increment or decrement certainly require a value, but there are also actions that make sense on accessible widgets that clearly don't have a value, such as scrolling actions on a scrollview, or showing/hiding of a tooltip.

```suggestion
            } else if (t == QAccessible::ActionInterface) {
```

Reply:
indeed.  bad copy paste.